content/03.results.md

```diff
@@ -4,10 +4,32 @@
 
 We previously performed whole genome sequencing (WGS), whole exome sequencing (WXS), and RNA sequencing (RNA-Seq) on matched tumor and normal tissues as well as selected cell lines from 943 patient tumors from the Pediatric Brain Tumor Atlas (PBTA) [@doi:10.1093/neuonc/noz192], consisting of samples from the Children’s Brain Tumor Network (CBTN) [@url:https://CBTN.org] and the PNOC003 DMG clinical trial [@doi:10.1002/ijc.32258] of the Pacific Pediatric Neuro-Oncology Symposium (PNOC) (**Figure {@fig:Fig1}A**). 
 We then harnessed the benchmarking efforts of the Kids First Data Resource Center to develop a robust and reproducible data analysis workflow within the CAVATICA platform to perform the following primary somatic analyses: variant calling of single nucleotide variants (SNVs), copy number variants (CNVs), structural variants (SVs), and fusions (**Figure S1** - red boxes and **STAR Methods**). 
-Next, we created a public Github analysis repository (https://github.com/AlexsLemonade/OpenPBTA-analysis) with continuous integration to ensure analysis reproducibility.
-We additionally created a GitHub manuscript repository (https://github.com/AlexsLemonade/OpenPBTA-manuscript) with ManuBot [@url:https://manubot.org/ ; @doi:10.1371/journal.pcbi.1007128] integration to enable manuscript creation using Markdown within GitHub.
-
-We maintained a data release folder on Amazon S3 containing merged files for each analysis, downloadable from the GitHub repository or open access CAVATICA project.  
+We maintained a data release folder on Amazon S3 containing merged files for each analysis, downloadable from the GitHub repository or open access CAVATICA project.
+As new analytical products that we expected to be used across multiple analyses were produced (e.g., tumor mutation burden calculations) or issues with the data were identified, new releases were made available in a versioned manner.
+
+A key feature of this project was the open contribution model used for both analyses (i.e., analytical code) and scientific manuscript writing. 
```

Comment:
I like the addition of the word `innovative`. I am less sold on the reference to things being coupled. I don't think that's specific to this project – I expect many Manubot papers use version controlled figures, etc.

```suggestion
A key innovative feature of this project was the open contribution model used for both analyses (i.e., analytical code) and scientific manuscript writing. 
```

Reply:
Sure. I used "coupled" because I didn't think the open manubot writing itself isn't innovative (or maybe it still is since it hasn't been done _much_?).